Pcne-s4-hybrid-connectivity.md

```
@@ -0,0 +1,763 @@
+```mermaid
+flowchart TB
+    subgraph Metro["単一メトロ(例: Dallas)"]
+        Domain1["エッジ可用性ドメイン 1"]
+        Domain2["エッジ可用性ドメイン 2"]
+    end
+    OnPrem1[オンプレミス<br/>ルータ] --> Domain1
+    OnPrem1 --> Domain2
+    Domain1 --> Router1[Cloud Router]
+    Domain2 --> Router1
+    Router1 --> VPC1["VPC(us-central1)"]
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)